client/asset/doge/doge.go

```diff
@@ -0,0 +1,160 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package doge
+
+import (
+	"fmt"
+
+	"decred.org/dcrdex/client/asset"
+	"decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/dex"
+	dexbtc "decred.org/dcrdex/dex/networks/btc"
+	dexdoge "decred.org/dcrdex/dex/networks/doge"
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
+const (
+	version = 0
+	BipID   = 3
+	// The default fee is passed to the user as part of the asset.WalletInfo
+	// structure.
+	defaultFee          = 500000
```

Comment:
such fees, much zeros

Reply:
Seems too high for the fallback fee.  500,000 sats/byte = 5 DOGE /kB >> 0.01 DOGE / kB  https://github.com/dogecoin/dogecoin/blob/master/doc/fee-recommendation.md